Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -484,3 +484,34 @@ function Add-AzureStackAzureRmEnvironment {
         throw
     }
 }
+
+function Disconnect-AzureAndClearContext {
+    [CmdletBinding()]
+    param()
+
+    try {
+        Write-Verbose "Trying to disconnect from Azure and clear context"
+
+        if (Get-Command -Name "Disconnect-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Disconnect-AzureRmAccount"
+            Disconnect-AzureRmAccount
+        }
+        elseif (Get-Command -Name "Remove-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Remove-AzureRmAccount"
+            Remove-AzureRmAccount
+        }
+        elseif (Get-Command -Name "Logout-AzureRmAccount" -ErrorAction "SilentlyContinue") {
+            Write-Host "##[command]Logout-AzureRmAccount"
+            Logout-AzureRmAccount
+        }
```

Comment:
log a warning here as well saying unable find command to logout session

Reply:
Logged